sfge/src/main/java/com/salesforce/graph/symbols/apex/schema/DescribeSObjectResultFactory.java

```diff
@@ -106,33 +128,46 @@ public final class DescribeSObjectResultFactory {
                                         .returnedFrom(sObjectType, vertex)
                                         .buildDescribeSObjectResult(sObjectType));
                     }
-                } else if (SystemNames.DML_OBJECT_ACCESS_METHODS.contains(methodName)
-                        && chainedNames.size() == 3) {
+                } else if (SystemNames.DML_OBJECT_ACCESS_METHODS.contains(methodName)) {
                     // Schema.SObjectType.Account.isDeletable()
-                    if (chainedNames
+                    if (chainedNames.size() == 3
```

Comment:
This somehow feels redundant with the else-if at line 144. Could the if-condition be widened to include both cases and the code made flexible enough to cover both?

Reply:
Yeah, handling the `Schema.` prefix on some types and lack of it has been creating some redundancy. Will see how I can make this nicer.